### PR TITLE
Add definition for PVC Base and ECS cluster on AWS IaaS

### PR DESCRIPTION
**Pull request by jimright**

### CODE REVIEW

File: private-cloud/ecs/aws-iaas/roles/cluster_reqs/vars/main.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
```
Comment (wmudge) [inline private-cloud/ecs/aws-iaas/roles/cluster_reqs/vars/main.yml:15]: License...

File: private-cloud/ecs/aws-iaas/roles/krb_file_ticket_cache/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+---
```
Comment (wmudge) [inline private-cloud/ecs/aws-iaas/roles/krb_file_ticket_cache/defaults/main.yml:15]: License...

File: private-cloud/ecs/aws-iaas/roles/cluster_reqs/tests/test.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (wmudge) [inline private-cloud/ecs/aws-iaas/roles/cluster_reqs/tests/test.yml:15]: License...

File: private-cloud/ecs/aws-iaas/roles/cluster_reqs/tests/inventory [hunk 4]
```diff
@@ -0,0 +1,2 @@
+localhost
```
Comment (wmudge) [inline private-cloud/ecs/aws-iaas/roles/cluster_reqs/tests/inventory:15]: License...

File: private-cloud/ecs/aws-iaas/roles/cluster_reqs/meta/main.yml [hunk 5]
```diff
@@ -0,0 +1,52 @@
+galaxy_info:
```
Comment (wmudge) [inline private-cloud/ecs/aws-iaas/roles/cluster_reqs/meta/main.yml:15]: License...

File: private-cloud/ecs/aws-iaas/roles/cluster_reqs/defaults/main.yml [hunk 6]
```diff
@@ -0,0 +1,2 @@
+---
```
Comment (wmudge) [inline private-cloud/ecs/aws-iaas/roles/cluster_reqs/defaults/main.yml:15]: License...

File: private-cloud/ecs/aws-iaas/roles/krb_file_ticket_cache/handlers/main.yml [hunk 7]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (wmudge) [inline private-cloud/ecs/aws-iaas/roles/krb_file_ticket_cache/handlers/main.yml:15]: License...

File: private-cloud/ecs/aws-iaas/roles/krb_file_ticket_cache/tasks/main.yml [hunk 8]
```diff
@@ -0,0 +1,27 @@
+---
```
Comment (wmudge) [inline private-cloud/ecs/aws-iaas/roles/krb_file_ticket_cache/tasks/main.yml:15]: License

**Review by wmudge — CHANGES_REQUESTED** [review]

File: private-cloud/ecs/aws-iaas/roles/cluster_reqs/meta/main.yml (hunk 5 above)
Comment (jimright) [reply private-cloud/ecs/aws-iaas/roles/cluster_reqs/meta/main.yml:15]: Done in [85561d5](https://github.com/cloudera-labs/cloudera-deploy/pull/143/commits/85561d5db6137299e7e25d323b5761ba600d42e4)

File: private-cloud/ecs/aws-iaas/roles/cluster_reqs/vars/main.yml (hunk 1 above)
Comment (jimright) [reply private-cloud/ecs/aws-iaas/roles/cluster_reqs/vars/main.yml:15]: Done in [85561d5](https://github.com/cloudera-labs/cloudera-deploy/pull/143/commits/85561d5db6137299e7e25d323b5761ba600d42e4)

File: private-cloud/ecs/aws-iaas/roles/krb_file_ticket_cache/defaults/main.yml (hunk 2 above)
Comment (jimright) [reply private-cloud/ecs/aws-iaas/roles/krb_file_ticket_cache/defaults/main.yml:15]: Done in [85561d5](https://github.com/cloudera-labs/cloudera-deploy/pull/143/commits/85561d5db6137299e7e25d323b5761ba600d42e4)

File: private-cloud/ecs/aws-iaas/roles/cluster_reqs/tests/test.yml (hunk 3 above)
Comment (jimright) [reply private-cloud/ecs/aws-iaas/roles/cluster_reqs/tests/test.yml:15]: Done in [85561d5](https://github.com/cloudera-labs/cloudera-deploy/pull/143/commits/85561d5db6137299e7e25d323b5761ba600d42e4)

File: private-cloud/ecs/aws-iaas/roles/cluster_reqs/defaults/main.yml (hunk 6 above)
Comment (jimright) [reply private-cloud/ecs/aws-iaas/roles/cluster_reqs/defaults/main.yml:15]: Done in [85561d5](https://github.com/cloudera-labs/cloudera-deploy/pull/143/commits/85561d5db6137299e7e25d323b5761ba600d42e4)

File: private-cloud/ecs/aws-iaas/roles/krb_file_ticket_cache/tasks/main.yml (hunk 8 above)
Comment (jimright) [reply private-cloud/ecs/aws-iaas/roles/krb_file_ticket_cache/tasks/main.yml:15]: Done in [85561d5](https://github.com/cloudera-labs/cloudera-deploy/pull/143/commits/85561d5db6137299e7e25d323b5761ba600d42e4)

**Comment by rsuplina** [comment]
Tested the deployment, ✅ "works on my Machine"

**Comment by clevesque** [comment]
a few notes on this example:
Can we avoid hardcoding instance type, instance counts, disk sizes/types and ami types inside the terraform config files? Would like to have the option  parameterize those values and pass in from definition.yml
Would like to use a ElasticIP on the bastion, instead of transient public IP.
ty

**Review by wmudge — APPROVED** [review]

**Pull request merged**